plugins/chanbackup.c

```diff
@@ -160,60 +166,54 @@ static void maybe_create_new_scb(struct plugin *p,
 	close(fd);
 
 	/* This will update the scb file */
-	rename("scb.tmp", "emergency.recover");
+	rename("scb.tmp", FILENAME);
 }
 
+static u8 *get_file_data(const tal_t *ctx, struct plugin *p)
+{
+	u8 *scb = grab_file(ctx, "emergency.recover");
```

Comment:
I think we should use `FILENAME` here as well instead of the similar string. This way, when filename get changed, code does not break.

Reply:
Agreed. Thank you for the review!